spec/support/libraryStubData.js

```diff
@@ -44,7 +56,15 @@ var libraryStubData = {
     'partner.js': partnerStub,
     'openrtb.js': openRtbStub,
     'size.js': {
-        arrayToString: function (arr) {
+        arrayToString: function (arr, delim) {
```

Comment:
There are 2 different signatures for this function in use. Updated the mock function for both cases.
https://github.com/indexexchange/open-x-adapter/blob/master-v2/open-x-htb.js#L125
https://github.com/indexexchange/open-x-adapter/blob/master-v2/open-x-htb.js#L315